### DISCUSSION TITLE
Option to send STATUS as a retained MQTT message

### DISCUSSION THREAD

**Discussion by jeroenst** [arendst/Tasmota#16079] [Ideas]
@arendst Is it possible to reconsider https://github.com/arendst/Tasmota/pull/11109 ?
I also see that other users are wanting this feature ( https://github.com/arendst/Tasmota/issues/2411 ).

I use these status messages for my status dashboard which uses MQTT websockets, but I now have build a nodered function to create retained status messages from the volatile status messages, this works but is kind a workaround.

![image](https://user-images.githubusercontent.com/2833940/180841586-c2a4bdd3-0dcc-444a-8440-9d4c51a27846.png)

**Comment by jeroenst** [comment]
@arendst Can I make a define for this feature so it doesn't eat memory when not used?

**Reply by arendst** [reply]
Yes

**Reply by sfromis** [reply]
Of course, quite a bit of device discovery info is by default also available in the retained `tasmota/discovery/.../config` messages.

**Reply by arendst** [reply]
I'll implement it now. @jeroenst No need to do anything,

**Reply by arendst** [reply]
Done

**Reply by jeroenst** [reply]
Thank you very much!
